### DISCUSSION TITLE
How to calculate the effect of shade inside arcades/colonnades and of similar architectural features

### DISCUSSION THREAD

**Discussion by oraleks** [UMEP-dev/UMEP#359] [Ideas]
Calculating shade from balconies, eaves or arcades can be useful, especially for analyzing locations where these features are consistently implemented as a heat mitigation strategy of urban design. This functionality might be integrated into UMEP by adding a third DSM to the building and tree canopy DSMs used for SVF and shade calculation.

**Comment by rarygit** [comment]
Perhaps you could adapt the DSM approach for a pergola as an arcade or colonnade feature attached to a building.

I provided a decription in either 'Discussions' or 'Issues' as to how I do this, using DSM generator and the Serval plugin to adjust DSM cell values:  https://github.com/lutraconsulting/serval/blob/master/Serval/docs/user_manual.md

Actually, it was here, in relation to green roofs: https://github.com/UMEP-dev/UMEP/discussions/328

 For colonnades etc, you will need to check whether the shading is correct. 
Features such as loggia would not be a problem because the "crown height" i.e. mass above the ceiling of the loggia, is higher than 1 m.

**Comment by biglimp** [comment]
Adding a new raster locating the overhangs etc potentially possible. If we get time we maybe will implement it but it is not priority at the moment.
